Sentry.Xamarin.Forms/SentryXamarinFormsIntegration.cs

```diff
@@ -114,6 +136,9 @@ private async Task<Application> GetCurrentApplication()
             return Application.Current;
         }
 
+        #endregion
+        #region Events
```

Comment:
😱 
Use of `#region` merged to _main_ is 5 to 10 years of solitaire confinement.